### PR TITLE
snap-bootstrap: add bind option with tests

### PR DESCRIPTION
**Pull request by xnox**
Add new `Bind` option to systemdMountOptions struct.

### CODE REVIEW

File: cmd/snap-bootstrap/initramfs_systemd_mount.go [hunk 1]
```diff
@@ -69,6 +70,8 @@ type systemdMountOptions struct {
 	// NoSuid indicates that the partition should be mounted with nosuid set on
 	// it to prevent suid execution.
 	NoSuid bool
+	// Bind this is a bind-mount
```
Comment (anonymouse64) [inline cmd/snap-bootstrap/initramfs_systemd_mount.go:73]:
```suggestion
	// Bind indicates this is a bind-mount
```

**Review by anonymouse64 — APPROVED** [review]
one nitpick on the doc-comment, but otherwise lgtm

Careful, we might just make you into a real Go developer yet :smile:

**Comment by xnox** [comment]
2 required spreads have failed, and I don't know why. Or what I can do to fix them.

**Comment by anonymouse64** [comment]
@xnox the spread failures here are unrelated to your change and can be ignored. On Monday when Samuele and Michael are back, one of them can force merge the PR with the failures still here

**Review by stolowski — APPROVED** [review]
LGTM

**Comment by xnox** [comment]
@mvo5 @pedronis merge failures appear to be unrelated to the code changed. Please merge?

**Pull request merged**